### PR TITLE
Remove nocast branching from bigquery hook get_records method

### PR DESCRIPTION
**Pull request by pankajastro**
Remove serialize vs non serialize branching logic in BigQueryGetOperatorAsync

Closes: #270

### CODE REVIEW

**Review by phanikumv — CHANGES_REQUESTED** [review]
you won't need the `nocast` variable anymore in `get_records` method argument, if you remove the branching logic.

File: astronomer/providers/google/cloud/hooks/bigquery.py [hunk 1]
```diff
@@ -132,14 +132,9 @@ def get_records(self, query_results: Dict[str, Any], nocast: bool = True) -> Lis
         """
         buffer = []
         if "rows" in query_results and query_results["rows"]:
-            fields = query_results["schema"]["fields"]
```
Comment (phanikumv) [inline astronomer/providers/google/cloud/hooks/bigquery.py:135]: Remove param `nocast` as well please

Comment (pankajastro) [reply astronomer/providers/google/cloud/hooks/bigquery.py:135]: Removed

**Comment by pankajastro** [comment]
> you won't need the `nocast` variable anymore in `get_records` method argument, if you remove the branching logic.

yes, removed it.

**Review by phanikumv — APPROVED** [review]

**Pull request merged**